Dockerfile

```diff
@@ -60,8 +64,8 @@ ENV PATH /edx/app/edx-platform/edx-platform/bin:${PATH}
 ENV SETTINGS production
 
 # TODO: Install requirements before copying in code.
-RUN pip install setuptools==39.0.1 pip==9.0.3
-RUN pip install -r requirements/edx/base.txt
+RUN python3.8 -m pip install setuptools==39.0.1 pip==9.0.3
```

Comment:
These are too old, we've already updated production to use setuptools 44.1.0 and pip 20.0.2.